### PR TITLE
chore(dot-org): use walletconnect.org for the relay and rpc urls

### PR DESCRIPTION
**Pull request by Cali93**
## Description

We should use walletconnect.org everywhere.

## Type of change

- [X] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)

## How has this been tested?

Covered by existing tests

## Fixes/Resolves (Optional)

Users in some regions getting blocked by their ISPs

## Checklist

- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ganchoradkov — APPROVED** [review]
💯

File: packages/utils/test/cacao.spec.ts [hunk 1]
```diff
@@ -22,7 +19,7 @@ describe("URI", () => {
     it("should merge recaps", () => {
       const recap1 = {
         att: {
-          "https://notify.walletconnect.com": { "manage/all-apps-notifications": [{}] },
```
Comment (ganchoradkov) [inline packages/utils/test/cacao.spec.ts]: btw these updates might case tests to fail

**Review by ganchoradkov — APPROVED** [review]

File: packages/utils/test/cacao.spec.ts (hunk 1 above)
Comment (Cali93) [reply packages/utils/test/cacao.spec.ts]: Curious why ? I checked that notify server also runs under notify.walletconnect.org

Comment (Cali93) [reply packages/utils/test/cacao.spec.ts]: Ah I just noticed it's because the recap test includes a JWT with the old url. Will revert these back then

**Review by bkrem — APPROVED** [review]
🙏

**Pull request merged**